packages/botonic-cli/src/commands/destroy.ts

```diff
@@ -0,0 +1,46 @@
+import { PulumiRunner } from '@botonic/pulumi/lib/pulumi-runner'
+import { Command } from '@oclif/command'
+
+import { Telemetry } from '../analytics/telemetry'
+import { CLOUD_PROVIDERS, PATH_TO_AWS_CONFIG } from '../constants'
+
+export default class Run extends Command {
+  static description = 'Deploy Botonic project to hubtype.com'
+
+  static examples = [
+    `$ botonic destroy aws
+Destroying AWS stack...
+`,
+  ]
+  static flags = {}
+
+  static args = [{ name: 'provider', options: Object.values(CLOUD_PROVIDERS) }]
+
+  private telemetry = new Telemetry()
+
+  /* istanbul ignore next */
+  async run(): Promise<void> {
+    const { args } = this.parse(Run)
+    const provider = args.provider
+    if (provider) {
+      this.destroyProvider(provider)
+    }
+  }
+
+  async destroyProvider(provider: string): Promise<void> {
+    this.telemetry.trackDeploy({ provider })
+    if (provider === CLOUD_PROVIDERS.AWS) {
```

Comment:
Same here, what happens if the developer sets a wrong provider or a provider that is not supported right now? Should we show a message?

Reply:
good point! in fact in the `args` member of `Run` class (from occlif) you can set options, which is set with `Object.values(CLOUD_PROVIDERS)`. If the deploy command is called with an argument not in options, the following message will appear:
```
yarn run v1.22.10
$ yarn bt deploy foo
$ /Users/mrabat/Documents/hubtype/botonic/packages/botonic-cli/bin/run deploy foo
 ›   Error: Expected foo to be one of: aws, hubtype
```
as this is resolved as soon as the command is executed, I found it not necessary to add more info in latter steps of the flow.

Reply:
Thanks for the explanation! Totally agree, no need to add more messages.